README.md

```diff
@@ -1,124 +1,95 @@
+[![Published on NPM](https://img.shields.io/npm/v/@polymer/iron-a11y-keys.svg)](https://www.npmjs.com/package/@polymer/iron-a11y-keys)
 [![Build status](https://travis-ci.org/PolymerElements/iron-a11y-keys.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-a11y-keys)
-
-_[Demo and API docs](https://elements.polymer-project.org/elements/iron-a11y-keys)_
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/iron-a11y-keys)
 
 ## &lt;iron-a11y-keys&gt;
-
 `iron-a11y-keys` provides a cross-browser interface for processing
 keyboard commands. The interface adheres to [WAI-ARIA best
 practices](http://www.w3.org/TR/wai-aria-practices/#kbd_general_binding).
 It uses an expressive syntax to filter key presses.
 
-## Basic usage
+See: [Documentation](https://www.webcomponents.org/element/@polymer/iron-a11y-keys),
+  [Demo](https://www.webcomponents.org/element/@polymer/iron-a11y-keys/demo/demo/index.html).
 
-The sample code below is a portion of a custom element. The goal is to call
-the `onEnter` method whenever the `paper-input` element is in focus and
-the `Enter` key is pressed.
+## Usage
 
-```html
-<iron-a11y-keys id="a11y" target="[[target]]" keys="enter"
-                    on-keys-pressed="onEnter"></iron-a11y-keys>
-<paper-input id="input"
-             placeholder="Type something. Press enter. Check console."
-             value="{{userInput::input}}"></paper-input>
+### Installation
+```
+npm install --save @polymer/iron-a11y-keys
 ```
 
-The custom element declares an `iron-a11y-keys` element that is bound to a
-property called `target`. The `target` property
-needs to evaluate to the `paper-input` node. `iron-a11y-keys` registers
-an event handler for the target node using Polymer's [annotated event handler
-syntax](https://www.polymer-project.org/1.0/docs/devguide/events.html#annotated-listeners). `{{userInput::input}}` sets the `userInput` property to the
-user's input on each keystroke.
-
-The last step is to link the two elements within the custom element's
-registration.
-
-```javascript
-...
-properties: {
-  userInput: {
-    type: String,
-    notify: true,
-  },
-  target: {
-    type: Object
-  },
-},
-ready: function() {
-  this.target = this.$.input;
-},
-onEnter: function() {
-  console.log(this.userInput);
+### In an html file
+```html
+<html>
+  <head>
+    <script type="module">
+      import '@polymer/iron-a11y-keys/iron-a11y-keys.js';
+    </script>
+  </head>
+  <body>
+    <iron-a11y-keys id="keys" keys="left right down up"></iron-a11y-keys>
+  </body>
+  <script>
```

Comment:
This should probably go in either `head` or `body`.

Reply:
it doesn't matter (the browser moves it), but ill move it to the body.